packages/client/hmi-client/src/components/workflow/ops/tera-node-charts.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+	<tera-progress-spinner v-if="isLoading" :font-size="2" is-centered style="height: 100%">
+		<div v-if="processing">{{ processing }}</div>
+		<div v-else>Processing...</div>
+	</tera-progress-spinner>
+	<div v-else-if="visibleChartSettings.length">
+		<div v-for="setting of visibleChartSettings" :key="chartSettingKey || setting.id">
+			<vega-chart
+				:expandable="expandable"
+				:are-embed-actions-visible="areEmbedActionsVisible"
+				:visualization-spec="preparedCharts[chartSettingKey || setting.id]"
```

Comment:
Sorry, to clarify by earlier comment.  `interactive` prop is an opt-out mechanism - if you don't explicitly set to false the charts will be interactive. You can verify this by looking at simulate charts which has tooltip interactions if interactive mode is on.

<img width="347" alt="image" src="https://github.com/user-attachments/assets/0752866a-688b-4626-861a-e70ba1e7f7fe" />

For the nodes pascale had asked the interactions to be turned off (to save memory, as we would render to png/svg and dispose the vegalite context).